content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,634 @@
+loads. Depending on the rights of the user, changes can be made. To modify the request, click the
+:guilabel:`Edit` button, make any desired changes, then click :guilabel:`Save`. It is also possible
+to approve or refuse the request from this form. Click the :guilabel:`Validate` button to approve,
+or the :guilabel:`Refuse` button to refuse the request.
+
+.. _time_off/approve-time-off:
+
+Approve time off
+----------------
+
+To view time off requests that need approval, navigate to :menuselection:`Time Off application -->
+Approvals --> Time Off`. The only time off requests visible on this list are for employees the user
+has either :guilabel:`Time Off Officer` or :guilabel:`Administrator` access rights for the *Time
+Off* application.
+
+The default filters in the :guilabel:`Time Off` list are :guilabel:`To Approve`, :guilabel:`My
+Team`, :guilabel:`Active Employee`, and :guilabel:`Active Time Off`. This presents only time off
+requests that need to be approved for current employees on the user's team, for requests that are
+active and not in a draft mode.
+
+The left side of the screen has various grouping options to narrow down the presented time off
+requests. Since only time off requests that need to be approved are shown, the only status options
+are :guilabel:`All` and :guilabel:`To Approve`. To view requests with other statuses, first remove
+the :guilabel:`To Approve` filter. Click the X on the :guilabel:`To Approve` filter to remove it.
+
+To display time off requests for specific departments, click on the department on the left hand
+side. Only requests within the selected department will be presented.
+
+The status column displays the status of each request, with the status highlighted in a specific
+color. The :guilabel:`To Approve` requests are highlighted in yellow, and are the only ones that
+appear in the list by default. If the :guilabel:`To Approve` filter is removed, then all statuses
+appear. :guilabel:`Approved` requests are highlighted in green, :guilabel:`To Submit` (drafts)
+requests are highlighted in blue, and the :guilabel:`Refused` requests are highlighted in gray.
+
+To approve a time off request, click :guilabel:`👍 Approve` at the end of the line, to refuse a
+request, click :guilabel:`X Refuse`.
+
+.. image:: time_off/time-off-requests.png
+   :align: center
+   :alt: Time off requests with the filter, groupings, and status sections highlighted.
+
+If more details are needed, click anywhere on the time off request line (except for :guilabel:`👍
+Approve` and :guilabel:`X Refuse`) to load the time off request form. Depending on the rights of the
+user, changes can be made. To modify the request, click the :guilabel:`Edit` button, make any
+desired changes, then click :guilabel:`Save`. It is also possible to approve or refuse the request
+from this form. Click the :guilabel:`Approve` button to approve, or the :guilabel:`Refuse` button to
+refuse the request.
+
+My time off
```

Comment:
```suggestion
My Time Off
```

Reply:
figured since you've capitalized actual sections/pages like this in headers before, I figured this could follow suit 👍

Reply:
whichever route you choose, just make sure to keep everything uniform

Reply:
I'll go with all lower case, since the majority already is.